commitlint.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  rules: {
+    "type-case": [2, "always", "lower-case"],
+    "type-empty": [2, "never"],
+    "type-enum": [2, "always", ["feat", "fix", "docs", "build", "chore"]],
```

Comment:
fair point, i am one of those dudes who use chore for stuff which I cannot define otherwise :grin: I'd like to keep it because otherwise fix: will be my new goto then and thats more misleading. :sweat_smile:

Reply:
however i like adding refactor and ci :)

Reply:
done :heavy_check_mark: